ignite/templates/app/stargate/cmd/{{binaryNamePrefix}}d/cmd/config.go.plush

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"<%= ModulePath %>/app"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func initSDKConfig() {
```

Comment:
Intially this method was called `SetPrefixes`, but since it seals the config, I found this was appropriate to name it `initSDKConfig`. There's a similar method in `osmosis` for instance.